app/views/sessions/_form.html.erb

```diff
@@ -1,24 +1,32 @@
 <%= form_for :session, url: session_path do |form| %>
-  <div class="text-field">
-    <%= form.label :email %>
-    <%= form.text_field :email, type: 'email', class: "input-field" %>
-  </div>
+  <div class="flex flex-col items-center px-4 py-2 md:px-32 lg:px-60 py-10 lg:py-14">
+    <div class="content-center text-center py-4 md:w-max mt-2.5"> 
+      <%= render "shared/header_stamp" %>
+      <h2 class="header-2 w-80 md:w-11/12 md:ml-12">Para continuar inicia sesión en tribo</h2>
+    </div>  
+      
+      <div class="text-field mt-8 md:w-full">
+        <%= form.text_field :email, type: 'email', class: "input-field w-60 md:w-full", placeholder: "Correo electrónico" %>
+      </div>
```

Comment:
You also noticed that they already include the login commits in this PR. 

![imagen](https://user-images.githubusercontent.com/12243331/108447861-4365c980-7226-11eb-92c7-64d64726a66f.png)